python/cuopt/cuopt/linear_programming/problem.py

```
@@ -1186,6 +1460,23 @@ def getCSR(self):
+        Qcsr_dict = {"row_pointers": [0], "column_indices": [], "values": []}
+        for i, var2 in enumerate(vars2):
+            row_dict = Qdict[var2.index] if var2.index in Qdict else {}
+            var1 = vars1[i]
+            coeff = coeffs[i]
+            row_dict[var1.index] = row_dict[var1.index] + coeff if var1.index in row_dict else coeff
+        for i in range(0, self.NumVariables):
+            Qcsr_dict["column_indices"].extend(
+                list(Qdict[i].keys())
+            )
+            Qcsr_dict["values"].extend(list(Qdict[i].values()))
+            Qcsr_dict["row_pointers"].append(len(csr_dict["column_indices"]))
```

Flake8 (7.3.0)

[error] undefined name 'csr_dict'

(F821)